environment_and_metadata.rst

```diff
@@ -184,18 +183,18 @@ environment variables for correct operation of most software.
 
 .. code::
 
-   $ singularity exec --cleanenv env.sif env
+   $ apptainer exec --cleanenv env.sif env
    HOME=/home/dave
    LANG=C
-   LD_LIBRARY_PATH=/.singularity.d/libs
+   LD_LIBRARY_PATH=/.apptainer.d/libs
```

Comment:
```suggestion
   LD_LIBRARY_PATH=/.singularity.d/libs
```
This path should remain unchanged because we are preserving the `/.singularity.d` directory in container root filesystems for image compatability with singularity